DFC.App.MatchSkills/Controllers/WorkedController.cs

```diff
@@ -18,25 +18,45 @@ public class WorkedController : CompositeSessionController<WorkedCompositeViewMo
             : base(compositeSettings, sessionService, cookieService)
         {
         }
-        
-        [HttpPost]
-        [Route("MatchSkills/[controller]")]
-        public async Task<IActionResult> Body(WorkedBefore choice)
+
+        public override async Task<IActionResult> Body()
         {
+
             var primaryKeyFromCookie = TryGetPrimaryKey(this.Request);
-            var userWorkedBefore = choice == WorkedBefore.Undefined ? (bool?)null : choice == WorkedBefore.Yes;
 
 
-            if (!string.IsNullOrWhiteSpace(primaryKeyFromCookie))
+            if (string.IsNullOrWhiteSpace(primaryKeyFromCookie))
             {
                 var createSessionRequest = new CreateSessionRequest()
                 {
-                    CurrentPage = CompositeViewModel.PageId.Worked.Value,
-                    UserHasWorkedBefore = userWorkedBefore
+                    CurrentPage = CompositeViewModel.PageId.Worked.Value
```

Comment:
I think you can just use ViewModel.PageId here?  Saves having to use the hard coded constant